salt/modules/debian_ip.py

```diff
@@ -679,13 +679,14 @@ def _parse_interfaces(interface_files=None):
 
     # Return a sorted list of the keys for bond, bridge and ethtool options to
     # ensure a consistent order
+    malformed_interfaces = set()
     for iface_name in adapters:
```

Comment:
**suggestion** This could actually just be:

```suggestion
    for iface_name in list(adapters):
```

Rather than iterating directly over adapters, `list` creates a copy of the keys in the OrderedDict.

Reply:
(obviously(?) it removes the need for any other changes)